src/main.js

```diff
@@ -54,7 +54,13 @@ export let createProject = async (options) => {
   options.templateDirectory = templateDir;
 
   try {
-    await access(templateDir, fs.constants.R_OK);
+    await access(templateDir, fs.constants.R_OK).then(_ => {
```

Comment:
Lovely solution there. Thank you. Please just always update the testing checklist, so that I run into less issues. Thanks.